build/version.cake

```diff
@@ -14,7 +14,7 @@ public class BuildVersion
         var vsixVersion = gitVersion.MajorMinorPatch;
```

Comment:
Why are we using `LegacySemVer` here, @arturcic? Shouldn't we be able to just use `FullSemVer` for the `semVersion` variable and rely on GitVersion's built-in logic of adding metadata, etc.? As far as I can tell, had we done that, #1717 shouldn't have happened in the first place.

Reply:
I think the reason I used the `LegacySemVer` instead of `FullSemVer` is because the same variable is used for the `gem` file, and if I'm not wrong there were some issues when a gem with that version was pushed to the respective repository. So for now I would prefer to use the `.` and hopefully the build does not break. I'll merge it as is now.